h/search/index.py

```diff
@@ -118,17 +118,16 @@ def index(self, annotation_ids=None):
         :rtype: set
         """
         if not annotation_ids:
-            annotations = _all_annotations(session=self.session,
-                                           windowsize=PG_WINDOW_SIZE)
+            annotations = _all_annotations(session=self.session, windowsize=windowsize)
         else:
             annotations = _filtered_annotations(session=self.session,
                                                 ids=annotation_ids)
 
         # Report indexing status as we go
-        annotations = _log_status(annotations, log_every=PG_WINDOW_SIZE)
+        annotations = _log_status(annotations, windowsize)
```

Comment:
I found it slightly clearer to read when the `log_every` argument was explicitly named rather than just relying on the position.